src/api/Areas/Dashboard/Controllers/ConfigurationItemController.cs

```diff
@@ -25,6 +29,7 @@ public class ConfigurationItemController : ControllerBase
     #region Variables
     private readonly ILogger _logger;
     private readonly IConfigurationItemService _service;
+    private readonly IAuthorizationHelper _authorization;
```

Comment:
Each controller has similar functionality.  They all check if the user has the `hsb` role.  If the user does not it will generate a query that only returns items they are allowed to see.